tests/phpunit/api/v3/ACLPermissionTest.php

```diff
@@ -664,4 +664,15 @@ protected function getActivityContacts($activity) {
     return $contacts;
   }
 
+  /**
+   * Test that the 'everyone' group can be given access to a contact.
+   */
+  public function testGetACLEveryonePermittedEntity() {
+    $this->setupScenarioCoreACLEveryonePermittedToGroup();
+    $this->callAPISuccess('Contact', 'getsingle', [
+      'id' => $this->scenarioIDs['Contact']['permitted_contact'],
+      'check_permissions' => 1,
+    ]);
```

Comment:
@eileenmcnaughton how about adding this is in
```suggestion
    ]);
    // Check that only permitted contacts can be seen
    $nonPermissionResult = $this->callAPISuccess('Contact', 'get', [
      'id' => $this->scenarioIDs['Contact']['non_permitted_contact'],
      'check_permissions' => 1,
    ]);
    $this->assertTrue(empty($nonPermissionResult['values']));
```

Reply:
good idea